### PR TITLE
Sorts cm_aliens, cm_marines, removes the last almayer reference

### PR DESCRIPTION
**Pull request by DominikPanic**
I just sorted the code, there was no cleanup done.

### CODE REVIEW

File: code/modules/shuttles/shuttle.dm [hunk 1]
```diff
@@ -186,3 +186,501 @@
 	for(var/obj/structure/engine_startup_sound/O in get_area(src))
 		playsound(O.loc, 'sound/effects/engine_startup.ogg', 75, 1)
 */
+
+/*************************************
+shuttle_backend.dm
```
Comment (Alterist) [inline code/modules/shuttles/shuttle.dm]: was this supposed to be removed already?

**Comment by Rohesie** [comment]
Merge conflict.

File: code/modules/shuttles/shuttle.dm (hunk 1 above)
Comment (spookydonut) [reply code/modules/shuttles/shuttle.dm]: dont mess with shuttle stuff pls

**Comment by spookydonut** [comment]
DNM because map changes are going to cause conflicts.

File: code/modules/shuttles/shuttle.dm (hunk 1 above)
Comment (DominikPanic) [reply code/modules/shuttles/shuttle.dm]: It's only moving a file, kind of necessary to get rid of cm_marines

**Comment by DominikPanic** [comment]
No point in me solving conflicts till map freeze is over.

**Pull request merged**